webview/src/experiments/components/table/TableFooter.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { AddAndRemoveBranches } from './commitsAndBranches/AddAndRemoveBranches'
+import styles from './styles.module.scss'
+
+export const TableFooter: React.FC = () => (
+  <tfoot className={styles.footer}>
```

Comment:
On a screen with a table shorter than the screen, this footer is difficult to spot:

<img width="1442" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/49e1619f-6ae5-4669-9d3b-c3639b191a77">

We discussed the footer being at the bottom of the table instead in planning but I forgot exactly what conclusion was reached 🤔 Other possible ideas to make it stand out more:

* add a border
* add a box shadow like we do with our sticky header and column
* change the background